scripts/asm-installer/install_asm

```diff
@@ -2751,6 +2755,18 @@ install_managed_components() {
   if [[ "$DISABLE_CANONICAL_SERVICE" -eq 0 ]]; then
     install_canonical_controller
   fi
+
+  info "Configuring ASM managed control plane revision CRD..."
+  kubectl apply -f "${CRD_CONTROL_PLANE_REVISION}"
+
+  info "Configuring ASM managed control plane revision CR for regular channel..."
+  kubectl apply -f "${CR_CONTROL_PLANE_REVISION_REGULAR}"
```

Comment:
Please prefix all of the `kubectl` statements with `run` since they modify the cluster

Reply:
Done